### PR TITLE
Add Digital-Currency

### PR DESCRIPTION
**Pull request by Icesofty**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [x] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [x] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Do you know if the ip of the demo changes anytime soon? A domain would be better.

**Comment by Icesofty** [comment]
> Do you know if the ip of the demo changes anytime soon? A domain would be better.

Hey, for know I don't think I'm going to buy a domain name only for the Demo website :) This can change later. 
I changed the default port, you can know directly access the demo http://64.227.62.141/

**Comment by nodiscc** [comment]
You can grab a free subdomain name from the excellent https://freedns.afraid.org/ community.

**Comment by Icesofty** [comment]
> You can grab a free subdomain name from the excellent https://freedns.afraid.org/ community.

Well, that was fast and easy, indeed. I learned something new today! Demo domain update for something more 'human'. Thanks

File: README.md [hunk 1]
```diff
@@ -991,6 +991,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 - [Cachet](https://cachethq.io/) - An open source status page system for everyone. ([Demo](https://demo.cachethq.io/), [Source Code](https://github.com/CachetHQ/Cachet)) `BSD-3-Clause` `PHP`
 - [CUPS](https://www.cups.org/) - The Common Unix Print System uses Internet Printing Protocol (IPP) to support printing to local and network printers. ([Source Code](https://github.com/apple/cups)) `GPL-2.0` `C`
 - [CyberChef](https://github.com/gchq/CyberChef) - Perform all manner of "cyber" operations within a web browser such as AES, DES and Blowfish encryption and decryption, creating hexdumps, calculating hashes, and much more. ([Demo](https://gchq.github.io/CyberChef)) `Apache-2.0` `JavaScript`
+- [Digital-Currency](https://github.com/Icesofty/Digital-Currency#readme) - Create your own Self-Hosted Digital Currency. ([Demo](http://tonken.mooo.com/), [Source Code](https://github.com/Icesofty/Digital-Currency)) `GPL-3.0` `Nodejs`
```
Comment (n8225) [inline README.md]: Remove #readme from the end of the main link.
Remove the Source code link as it is a duplicate of the main link.

Comment (Icesofty) [reply README.md]: Roger that

**Pull request merged**